app/src/debug/res/raw/initial_experiments.json

```diff
@@ -81,6 +81,54 @@
           "proposedEnrollment": 7,
           "userFacingDescription": "Experiment to test the home screen configurations",
           "last_modified": 1621443780172
+      },
+      {
+          "slug": "feature-search-term-groups-android",
+          "appId": "org.mozilla.fenix",
+          "appName": "fenix",
+          "channel": "nightly",
+          "branches": [
+              {
+                  "slug": "enabled",
+                  "ratio": 0,
+                  "feature": {
+                      "enabled": true,
+                      "featureId": "search-term-groups"
```

Comment:
Question: should this featureId (`"search-term-groups"`) match ln 104 and 114?

Reply:
I'd also love it if someone could look at this whole addition to `initial_experiments.json`, especially lines 108+

Reply:
@jhugman @travis79 could help to clarify :)